AWS/legos/aws_release_elastic_ip/aws_release_elastic_ip.py

```diff
@@ -0,0 +1,52 @@
+##
+# Copyright (c) 2023 unSkript, Inc
+# All rights reserved.
+##
+from pydantic import BaseModel, Field
+from typing import Dict
+import pprint
+
+
+class InputSchema(BaseModel):
+    public_ip: str = Field(

```

Comment:
i think it would be good to have a lego, which takes an instanceID and publicIP and removes that IP from the instance and releases it.

Reply:
I will work on this action @amit-chandak-unskript

Reply:
@harshal-netserv still this lego is not implemented the way i asked. I asked to take instanceID and publicIP as input to the lego and release that IP.

Reply:
Hey @amit-chandak-unskript I am considering **instanceID and publicIP as input to the Lego and releasing that IP**. as separate Lego and working on that. this Lego is for the remediation of Unattached Elastic IPs.